code/_onclick/click.dm

```diff
@@ -315,6 +315,8 @@ if(selected_ability.target_flags & flagname){\
 		return;}}
 
 /mob/living/carbon/xenomorph/proc/ability_target(atom/A)
+	TARGET_FLAGS_MACRO(XABB_HUMAN_TARGET, /mob/living/carbon/human)
+	TARGET_FLAGS_MACRO(XABB_XENO_TARGET, /mob/living/carbon/xenomorph)
```

Comment:
Why this can just use XABB_MOB_TARGET

Reply:
It's not impossible than an incoucious human or xeno are on the same turf than a consiouc mob that you really want to target.

This probably happens rarely, but it's possible. Why not preventing that issue at least partly? Some abilities cannot target a human, let's try to find only xenos then

Reply:
Because it adds extra processing on every single ability use for an edge case that will basically never happen